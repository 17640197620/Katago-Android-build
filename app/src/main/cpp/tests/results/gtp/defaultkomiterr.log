: GTP Engine starting...
: KataGo v1.10.0
: Using koPOSITIONALscoreTERRITORYtaxNONEsui0 rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: hasButton = false
: Config override: koRule = POSITIONAL
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/defaultkomiterr.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: multiStoneSuicideLegal = false
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: rules = 
: Config override: scoringRule = TERRITORY
: Config override: searchRandSeed = forTesting
: Config override: taxRule = NONE
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . @ . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreTERRITORYtaxNONEsui0komi6.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 92
NN batches: 92
NN avg batch size: 1
PV: C16 D3 Q17 Q3
Tree:
: T   1.25c W   0.36c S   0.37c ( +0.2 L  +0.4) N     100  --  C16 D3 Q17 Q3
---Black(^)---
C16 : T   3.09c W   1.97c S   0.57c ( +0.5 L  +0.6) LCB   -0.35c P  7.47% WF  24.4 PSV      25 N      24  --  C16 D3 Q17 Q3
Q17 : T   1.61c W   0.65c S   0.47c ( +0.4 L  +0.5) LCB   -6.04c P  8.75% WF  12.0 PSV      11 N      12  --  Q17 D3 D17 Q3
Q3  : T   1.03c W   0.22c S   0.40c ( +0.3 L  +0.5) LCB   -6.95c P  8.95% WF  10.0 PSV       9 N      10  --  Q3 Q17 C4 D17
C4  : T   0.99c W  -0.01c S   0.33c ( +0.2 L  +0.3) LCB  -10.09c P  8.52% WF  10.0 PSV       8 N      10  --  C4 Q3 Q17 D17
D3  : T   1.07c W   0.24c S   0.38c ( +0.2 L  +0.5) LCB   -9.86c P  8.33% WF  10.0 PSV       8 N      10  --  D3 Q17 Q3 D17
R4  : T   0.34c W  -0.82c S   0.14c ( -0.1 L  +0.2) LCB  -19.10c P  9.08% WF   6.9 PSV       7 N       7  --  R4 Q17 C4 D17
D17 : T   0.63c W  -0.24c S   0.29c ( +0.1 L  +0.3) LCB  -20.84c P  7.53% WF   7.0 PSV       6 N       7  --  D17 D3 Q17 Q3
R16 : T   0.87c W   0.02c S   0.36c ( +0.2 L  +0.4) LCB  -20.66c P  7.10% WF   7.0 PSV       6 N       7  --  R16 D3 D17 Q3
Q4  : T  -0.62c W  -1.16c S   0.12c ( -0.1 L  +0.2) LCB  -88.46c P  7.11% WF   3.9 PSV       4 N       4  --  Q4 D3 Q17
D4  : T  -0.21c W  -0.75c S   0.13c ( -0.1 L  +0.3) LCB  -88.89c P  5.74% WF   4.0 PSV       4 N       4  --  D4 D17 Q17

: = Q17
: GPU -1 finishing, processed 92 rows 92 batches
: All cleaned up, quitting
