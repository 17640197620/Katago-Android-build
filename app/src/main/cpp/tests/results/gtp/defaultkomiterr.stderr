KataGo v1.10.0
Using koPOSITIONALscoreTERRITORYtaxNONEsui0 rules initially, unless GTP/GUI overrides this
Initializing board with boardXSize 19 boardYSize 19
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . @ . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreTERRITORYtaxNONEsui0komi6.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 92
NN batches: 92
NN avg batch size: 1
PV: C16 D3 Q17 Q3
Tree:
: T   1.25c W   0.36c S   0.37c ( +0.2 L  +0.4) N     100  --  C16 D3 Q17 Q3
---Black(^)---
C16 : T   3.09c W   1.97c S   0.57c ( +0.5 L  +0.6) LCB   -0.35c P  7.47% WF  24.4 PSV      25 N      24  --  C16 D3 Q17 Q3
Q17 : T   1.61c W   0.65c S   0.47c ( +0.4 L  +0.5) LCB   -6.04c P  8.75% WF  12.0 PSV      11 N      12  --  Q17 D3 D17 Q3
Q3  : T   1.03c W   0.22c S   0.40c ( +0.3 L  +0.5) LCB   -6.95c P  8.95% WF  10.0 PSV       9 N      10  --  Q3 Q17 C4 D17
C4  : T   0.99c W  -0.01c S   0.33c ( +0.2 L  +0.3) LCB  -10.09c P  8.52% WF  10.0 PSV       8 N      10  --  C4 Q3 Q17 D17
D3  : T   1.07c W   0.24c S   0.38c ( +0.2 L  +0.5) LCB   -9.86c P  8.33% WF  10.0 PSV       8 N      10  --  D3 Q17 Q3 D17
R4  : T   0.34c W  -0.82c S   0.14c ( -0.1 L  +0.2) LCB  -19.10c P  9.08% WF   6.9 PSV       7 N       7  --  R4 Q17 C4 D17
D17 : T   0.63c W  -0.24c S   0.29c ( +0.1 L  +0.3) LCB  -20.84c P  7.53% WF   7.0 PSV       6 N       7  --  D17 D3 Q17 Q3
R16 : T   0.87c W   0.02c S   0.36c ( +0.2 L  +0.4) LCB  -20.66c P  7.10% WF   7.0 PSV       6 N       7  --  R16 D3 D17 Q3
Q4  : T  -0.62c W  -1.16c S   0.12c ( -0.1 L  +0.2) LCB  -88.46c P  7.11% WF   3.9 PSV       4 N       4  --  Q4 D3 Q17
D4  : T  -0.21c W  -0.75c S   0.13c ( -0.1 L  +0.3) LCB  -88.89c P  5.74% WF   4.0 PSV       4 N       4  --  D4 D17 Q17
